tests/phpunit/TemplateTest.php

```diff
@@ -864,7 +864,7 @@ public function testJstorSICI() {
    $this->assertEquals('1961', $expanded->get('year'));
    $this->assertEquals('81', $expanded->get('volume'));
    $this->assertEquals('1', $expanded->get('issue'));
-   $this->assertEquals('43', $expanded->get('pages'));
+   $this->assertEquals('43–52', $expanded->get('pages'));
```

Comment:
This test is only designed to check that `43` is extracted from the SICI; if the metadata is found from other sources then the test no longer tests what it is meant to test (and thus needs re-thinking...) – perhaps `process_citation` needs replacing with a function that has a narrower scope.

Reply:
new test added